velox/type/Conversions.cpp

```diff
@@ -23,3 +23,35 @@ DEFINE_bool(
     " format of type conversions used for casting. This is a temporary solution"
     " that aims to facilitate a seamless transition for users who rely on the"
     " legacy behavior and hence can change in the future.");
+
+namespace facebook::velox::util {
+
+// This is based on Presto java's castToBoolean method.
+Expected<bool> castToBoolean(const char* data, size_t len) {
+  const auto& TU = static_cast<int (*)(int)>(std::toupper);
+
+  if (len == 1) {
+    auto character = TU(data[0]);
```

Comment:
Why don't we just call std::toupper(data[0]) directly?

Reply:
Its repeated a lot , this is shorter and easier to read imo.